runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoOverlay.java

```diff
@@ -60,26 +71,35 @@
 	private final OpponentInfoConfig opponentInfoConfig;
 	private final HiscoreManager hiscoreManager;
 	private final NPCManager npcManager;
+	private final ClientThread clientThread;
 
 	private Integer lastMaxHealth;
 	private int lastRatio = 0;
 	private int lastHealthScale = 0;
 	private String opponentName;
+	private int raidOpponentHealth = 0;
+	private int raidOpponentMaxHealth = 0;
+	private String raidOpponentName;
+	private final ScheduledExecutorService scheduler = Executors.newSingleThreadScheduledExecutor();
```

Comment:
SES should be injected, not instantiated here.

Reply:
With the new code, using varbits instead of the widget, I don't need the scheduler anymore, so I've removed it. Also all the formatting and initialization guidelines you pointed out are added in the latest commit.